Cpix/ContentKeyPeriodCollection.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Xml;
+using Axinom.Cpix.Internal;
+
+namespace Axinom.Cpix
+{
+	sealed class ContentKeyPeriodCollection : EntityCollection<ContentKeyPeriod>
+	{
+		public const string ContainerXmlElementName = "ContentKeyPeriodList";
+
+		public ContentKeyPeriodCollection(CpixDocument document) : base(document)
+		{
+		}
+
+		internal override string ContainerName => ContainerXmlElementName;
+
+		protected override XmlElement SerializeEntity(XmlDocument document, XmlNamespaceManager namespaces, XmlElement container, ContentKeyPeriod entity)
```

Comment:
I'm a little confused about this. This class name contains "Collection", but it seems like only a single entity is serialized here? Perhaps some comments could help? Same applies to Deserialise below.

Reply:
@Heronyme could you comment on this and then I'll approve.

Reply:
There can be many entities, these methods are call for each entity/element within the collection/xml. The collection in this case represents the "ContentKeyPeriodList" XML element, which can contain many "ContentKeyPeriod" elements. This class can also contain collection-level validation, if needed. Does this make sense?

Reply:
👍